Websites/perf.webkit.org/public/api/measurement-set.php

```diff
@@ -187,7 +196,10 @@ static function format_map()
     }
 
     private static function format_run(&$run, &$commit_time) {
-        $commit_time = intval($run['revision_time']);
+        # Value precision of 'epoch from ...' can vary between different versions of postgres.
+        # 1538041792670.999 will be 1538041792671 on Postgres 11 or earlier, and 1538041792670.999 on later versions.
+        # Round it up to keep compatibility.
```

Comment:
This comment would be even better if it explained why need to keep compatibility. Presumably this has to do either with other code breaking if we don't round up, or with existing content in database - but what exactly breaks if we don't?

Also, I'm a bit confused by the "up" part. Surely `.999` will be rounded up, but `.001` will be rounded down, if we get that?

Reply:
OK, this is intended to fix unit test added by https://commits.webkit.org/206368@main still works on newer version of Postgres. Will rephrase and provide another round down example.